src/curr/str.rs

```diff
@@ -253,14 +272,25 @@ impl core::str::FromStr for ScAddress {
 impl core::fmt::Display for ScAddress {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
         match self {
-            ScAddress::Account(a) => a.fmt(f)?,
+            ScAddress::Account(a) => a.fmt(f),
             ScAddress::Contract(Hash(h)) => {
                 let k = stellar_strkey::Contract(*h);
                 let s = k.to_string();
-                f.write_str(&s)?;
+                f.write_str(&s)
+            }
+            ScAddress::MuxedAccount(muxed_ed25519_account) => {
+                let k = stellar_strkey::Strkey::MuxedAccountEd25519(
+                    stellar_strkey::ed25519::MuxedAccount {
+                        ed25519: muxed_ed25519_account.ed25519.0.clone(),
```

Comment:
Clone shouldn't be necessary here I think because the type should be an array?

```suggestion
                        ed25519: muxed_ed25519_account.ed25519.0,
```

Reply:
Done